Wastewater_Seq_analysis/run_freyja_vrn_noBoot.sh

```diff
@@ -0,0 +1,85 @@
+#! /bin/bash
+# if any errors, then exit
+set -euxo pipefail
+
+###########################
+# Author: Pooja Gupta
+
+USAGE="
+Purpose:
+1) Run Freyja tool with wastewater sequencing data. 
+2) Retrieve BAM files for each sample generated by Viralrecon
+3) The input data are the bam files after ivar primer trimming step.
+Usage:
+run_freyja_vrn_noBoot.sh <wastewater sequencing run_name>
+Last updated on: June 13,2022
+"
+###########################
+
+echo $USAGE
+
+#Get Freyja tool (need to replace with Freyja's docker container)
+source /home/pgupta/miniconda3/etc/profile.d/conda.sh
+conda activate /home/pgupta/miniconda3/envs/freyja-lin
+
+run_name=$1
+analysis_dir=/Volumes/IDGenomics_NAS/wastewater_sequencing
+#creating required folders for analysis
+
+mkdir -p /Volumes/IDGenomics_NAS/wastewater_sequencing/$run_name/analysis/freyja/lineage_out
+workdir=$analysis_dir/$run_name/analysis/freyja
+in_dir=$analysis_dir/$run_name/analysis/viralrecon/variants/bowtie2  #using bam files from viralrecon instead of Cecret
+outdir=$workdir/lineage_out/
+results=$analysis_dir/$run_name/results
+
+scov2=$analysis_dir/covidseq_ref/MN908947.3.fasta
+
+echo "$(date): Starting Freyja" 
+echo "$(date): Getting the input varaints and depth file from bam files for Freyja analysis"
+for bam in ${in_dir}/*_trim.sorted.bam;
+do 
+    infile=${bam##*/}; echo "$(date): input file name is $infile"
+    sample=${infile%_UT*}; echo "$(date): base name is $sample" 
+    echo "$(date): Freyja running for $sample"  
+    freyja variants ${bam} --variants $workdir/${sample}_out_variants --depths $workdir/${sample}_out_depths --ref $scov2;
```

Comment:
This expects freyja to be in path, which requires conda.

We can eliminate conda with staphb's freyja container, which might might this easier to maintain